.github/workflows/serverless-verify.yaml

```diff
@@ -80,7 +80,7 @@ jobs:
             --port 443:443@loadbalancer
             --wait
       - name: run git auth tests
-        run: make -C hack/ci git-auth-test-on-cluster
+        run: make -C hack git-auth-test-on-cluster
```

Comment:
It will break this tests as `Makefile` is still in `ci` dir.

Reply:
Makefile moved